content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+
+The L7 proxies are called waypoints. Waypoints process L7 functions such as traffic routing, rich authorization policy
+enforcement, and enterprise-grade resilience. Waypoints run outside of your application deployments and can scale independently
+based on your needs, which could be for the entire namespace or for multiple services within the namespace. Compared with
+sidecars, you don’t need one waypoint per application pod, and you can scale your waypoint effectively based on its scope,
+thus saving significant amounts of CPU and memory in most cases.
+
+The separation between the L4 secure overlay layer and L7 processing layer allows incremental adoption of the ambient mode data
+plane in contrast to the earlier binary “all-in” injection of sidecars. Users can start with the secure overlay layer which
+offers mTLS with cryptographic identity, simple L4 authorization policy, and telemetry. Later on, complex L7 handling such as
+retries, traffic splitting, complex load balancing, and observability collection can be enabled on a case-by-case basis.
+
+## What is in the scope of the Beta?
+
+We recommend you explore the following Beta functions of ambient mode in production with appropriate caution, after validating
+them in test environments:
+
+- [Install](/docs/ambient/install/).
+- [Adding your workloads to the mesh](/docs/ambient/usage/add-workloads/) to gain mutual TLS with cryptographic identity, [L4 authorization policies](t/docs/ambient/usage/l4-policy/), and telemetry.
+- [Configure waypoints](/docs/ambient/usage/waypoint/) to [use L7 functions](/docs/ambient/usage/l7-features/) such as traffic shifting, request routing, rich authorization policy enforcement.
+- Istio ingress gateway can work with workloads in ambient mesh supporting all existing Istio APIs.
+- Use `istioctl` to operate waypoints, and troubleshoot ztunnel & waypoints.
+
+### Alpha features
+
+Other features we want to include in ambient mode have been implemented but remain in Alpha status in this release. Please help
+test them, so they can be promoted to Beta in 1.23 or later:
+
+- Multi-cluster installations
+- DNS proxying
+- Interoperability with sidecars
+- IPv6/Dual stack
+- SOCKS5 support (for outbound)
+- Istio’s classic APIs (`VirtualService` and `DestinationRule`)
+
+### Roadmap
+
+We have a number of features which are not yet implemented in ambient mode but are planned for upcoming releases:
+
+- Controlled egress traffic
+- Multi-network support
+- Improve `status` messages on resources to help troubleshoot and understand the mesh
+- VM support
+
+## What about sidecars?
+
+Sidecars are not going away, and remain first-class citizens in Istio. You can continue to use sidecars and they will remain
+fully supported.  For any feature outside of the Alpha or Beta scope for ambient mode, you should consider using the sidecar
+pattern before the feature is added to ambient mode. Some use cases, such as traffic shifting based on source labels, will
```

Comment:
The preliminary docs refer to "sidecar mode" as opposed to "sidecar pattern".  It might be good to use a consistent name to not confuse users.

I think "until" instead of "before".  (But that does imply that all sidecar features will be supported by ambient, which will likely not be true.)

Reply:
updated